src/plugins/controls/public/actions/edit_control_action.test.tsx

```diff
@@ -0,0 +1,136 @@
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { BehaviorSubject } from 'rxjs';
+
+import { coreMock } from '@kbn/core/public/mocks';
+import { dataPluginMock } from '@kbn/data-plugin/public/mocks';
+import { dataViewPluginMocks } from '@kbn/data-views-plugin/public/mocks';
+import dateMath from '@kbn/datemath';
+import { ErrorEmbeddable } from '@kbn/embeddable-plugin/public';
+import type { TimeRange } from '@kbn/es-query';
+import type { ViewMode } from '@kbn/presentation-publishing';
+
+import { getOptionsListControlFactory } from '../react_controls/controls/data_controls/options_list_control/get_options_list_control_factory';
+import type { OptionsListControlApi } from '../react_controls/controls/data_controls/options_list_control/types';
+import {
+  getMockedBuildApi,
+  getMockedControlGroupApi,
+} from '../react_controls/controls/mocks/control_mocks';
+import { getTimesliderControlFactory } from '../react_controls/controls/timeslider_control/get_timeslider_control_factory';
+import { EditControlAction } from './edit_control_action';
+
+const mockDataViews = dataViewPluginMocks.createStartContract();
+const mockCore = coreMock.createStart();
+const dataStartServiceMock = dataPluginMock.createStartContract();
+dataStartServiceMock.query.timefilter.timefilter.calculateBounds = (timeRange: TimeRange) => {
+  const now = new Date();
+  return {
+    min: dateMath.parse(timeRange.from, { forceNow: now }),
+    max: dateMath.parse(timeRange.to, { roundUp: true, forceNow: now }),
+  };
+};
+
+const dashboardApi = {
+  viewMode: new BehaviorSubject<ViewMode>('view'),
+};
+const controlGroupApi = getMockedControlGroupApi(dashboardApi, {
+  removePanel: jest.fn(),
+  replacePanel: jest.fn(),
+  addNewPanel: jest.fn(),
+  children$: new BehaviorSubject({}),
+});
+
+let optionsListApi: OptionsListControlApi;
+beforeAll(async () => {
+  const controlFactory = getOptionsListControlFactory({
+    core: mockCore,
+    data: dataStartServiceMock,
+    dataViews: mockDataViews,
+  });
+
+  const optionsListUuid = 'optionsListControl';
+  const optionsListControl = await controlFactory.buildControl(
+    {
+      dataViewId: 'test-data-view',
+      title: 'test',
+      fieldName: 'test-field',
+      width: 'medium',
+      grow: false,
+    },
+    getMockedBuildApi(optionsListUuid, controlFactory, controlGroupApi),
+    optionsListUuid,
+    controlGroupApi
+  );
+
+  optionsListApi = optionsListControl.api;
+});
+
+describe('Incompatible embeddables', () => {
+  test('Action is incompatible with Error Embeddables', async () => {
```

Comment:
Same comment, I think we can remove the ErrorEmbeddable test cases since they are no longer relevent.

Reply:
Also removed in 3917c17deee81a3071efbb8d5e40fb540583f1d4 :+1: